test/jdk/javax/swing/JComboBox/DisabledComboBoxFontTestAuto.java

```diff
@@ -44,13 +37,22 @@
 
 import static java.awt.image.BufferedImage.TYPE_INT_ARGB;
 
+/*
+ * @test
+ * @bug 7093691 8310072
+ * @summary Tests if JComboBox has correct font color when disabled/enabled
+ * @key headful
+ * @run main/othervm -Dsun.java2d.uiScale=1 DisabledComboBoxFontTestAuto
+ */
+
 public class DisabledComboBoxFontTestAuto {
     private static JComboBox combo, combo2;
     private static BufferedImage enabledImage, disabledImage, enabledImage2, disabledImage2;
     private static Path testDir;
     private static String lafName;
     private static StringBuffer failingLafs;
     private static int COMBO_HEIGHT, COMBO_WIDTH, COMBO2_HEIGHT, COMBO2_WIDTH;
+    private static final int TOLERANCE = 20;
 
     private static void createCombo() {
         combo = new JComboBox();
```

Comment:
If we can use unicode blank character instead of "Simple JComboBox" text. I think we make this test more robust and avoid tolerance checks.

Reply:
Updated.

Reply:
Can the initial bug "JComboBox has correct font color when disabled" be verified with this change?

Reply:
This change works fine with initial bug. Test failed if I comment out the fix proposed in initial bug for Nimbus LAF.